src/libraries/pretest.proj

```diff
@@ -94,7 +94,7 @@
           Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)' or '$(BuildTargetFramework)' == ''">
     <!-- Shared framework deps file generation. Produces a test shared-framework deps file. -->
     <GenerateTestSharedFrameworkDepsFile SharedFrameworkDirectory="$(NetCoreAppCurrentTestHostSharedFrameworkPath)"
-                                         RuntimeGraphFiles="$(RuntimeIdGraphDefinitionFile)"
+                                         RuntimeGraphFiles="$(SourceRuntimeIdentifierGraphFile)"
```

Comment:
In theory, as Platforms.csproj now runs as part of the inbox build, this task can use the generated runtime identifier instead. That said, I just left another comment suggesting to only use one property to avoid these differences.